vendor/assets/javascripts/mousetrap.js

```diff
@@ -575,8 +575,8 @@
                 //
                 // chrome will not fire a keypress if meta or control is down
                 // safari will fire a keypress if meta or meta+shift is down
-                // firefox will fire a keypress if meta, alt or control is down
-                if ((action == 'keypress' && !e.altKey && !e.metaKey && !e.ctrlKey) || _modifiersMatch(modifiers, callback.modifiers)) {
```

Comment:
@vinothkannans :wink: This is why we want to move to yarn to manage dependency so that we don't nuke custom fixes that we apply to vendored files. I think we should fork the repo and publish our own package here which includes the fix.

Reply:
absolutely, @pmusaraj can you get that project forked (under the discourse org) and then publish to npm or something?

Reply:
Done, using a branch in our fork of mousetrap in Github directly: 
https://github.com/discourse/mousetrap/tree/firefox-alt-ke y